analyze/analyses/garbage.rs

```diff
@@ -169,3 +170,14 @@ pub fn garbage(items: &ir::Items, opts: &opt::Garbage) -> Result<Box<traits::Emi
 
     Ok(Box::new(garbage_items) as Box<traits::Emit>)
 }
+
+pub(crate) fn get_unreachable_items<'a>(
```

Comment:
I moved this back into `garbage`, as mentioned in review. Good call! I think this makes a lot more sense :slightly_smiling_face: